docs/run.sh

```diff
@@ -209,7 +209,11 @@ BootstrapLinux() {
     #   https://stat.ethz.ch/pipermail/r-help//2012-September/335676.html
     # May 2020: we also need devscripts for checkbashism
     # Sep 2020: add bspm, remotes
-    Retry sudo apt-get install -y --no-install-recommends r-base-dev r-recommended qpdf devscripts r-cran-bspm r-cran-remotes
+    Retry sudo apt-get install -y --no-install-recommends r-base-dev r-recommended qpdf devscripts r-cran-remotes
+    
+    if [[ "${USE_BSPM}" != "FALSE" ]]; then
+        Retry sudo apt-get install -y --no-install-recommends r-cran-bspm
+    fi
```

Comment:
Unneeded complication.  The real use-or-not switch is 

```sh
    if [[ "${USE_BSPM}" != "FALSE" ]]; then
        echo "bspm::enable()" | sudo tee --append /etc/R/Rprofile.site >/dev/null
    fi
```

Reply:
OK. I get an error when trying to use R 3.5 on Ubuntu 18.04:

https://github.com/spatstat/spatstat.utils/runs/2319013549?check_suite_focus=true#step:3:246

It looked like it was related to BSPM so I tried to disable in this case, but I didn't succeed. Apart from switching to Ubuntu 18.04 and setting R_VERSION=3.5 is there other stuff I need to do to test this? Or is it really just a terrible idea to try to test R 3.5 with this setup?

Reply:
Or is the problem rather that R is still 4.0, and only the CRAN binary packages are 3.5 compatible?

https://github.com/spatstat/spatstat.utils/runs/2319013549?check_suite_focus=true#step:3:132

Reply:
Yes.

As I tried to say a few times, I do not run CI with older R versions.  The '3.5' is *not* a marker for R versions per se *but for the API change that required all packages to rebuild with R 3.5.0 and later* just how 4.0 marks the same thing R 4.0.0.  Hence we need _different Rutter PPAs_ and AFAIK only the 4.0 is updated.

Reply:
OK. Sorry for being slow here, but I don't understand how to use the environment variable R_VERSION=3.5 then. I have tried on both Ubuntu 18.04 and 16.04 with the same failure. Is there something else I can do with the current script or should I simply abandon the idea of testing on R 3.5 (other than doing it with docker as you said you do when it is needed)?

Reply:
If and when R changes, as it did recently with R 3.5 and R 4.0, and at least once before (with R 3.0, maybe?) then we need t be able to _switch wholesale_ between repo as you cannot mix them.

That is the while purpose of that variable: "what type of R repo".  Just how Michael's setup no longer supports Ubuntu 16.04 because, well, the day only has 24 hours and he has a real job too.  We cannot hit all targets at all times.

The real problem here may be a combination of insufficient documentation on my side, and an absolute willingness by you to imagine that more than one R version is supported by r-ci.  It may be, but I don't use it this way, and I think you got your wires crossed looking at the "wrong" environment variable and wishing for / imagining a different use than there is.

Reply:
Another part of the problem is that the variable is called `R_VERSION`, probably should have been called `R_API_VERSION` but cannot be changed now as it is used/driven from existing yaml files.

As for running with R 3.6.* or R 3.5.*, for starters we do not have an apt command to pull an older version.  (You can do gymastics with the Debian snapshot archive but I never tried that.)  Plus older versions are just that... and may bite with current libraries.  So I am content to farm this out to r-hub and other service and leave my CI use focused on current R.

Reply:
OK. I got it now. I just couldn't see the need for having the R_VERSION variable if switching it from the default value made things always fail. Anyway, thanks a lot for all the input. I'm on the phone now, but tomorrow I'll clean up the debugging messages so the PR is ready to go. You are if course also welcome to take it as it is and make whatever changes you like.

Reply:
So one TODO item is to improve the documentation, maybe you can make a suggestion.

A second, more ambitious one is to think about how older R could be running.  As per the previous comment, I do not see an obvious "quick" route.

Reply:
Please restore this to the line 207 above.